saturn-rapids/.env_deps

```diff
@@ -1,3 +1,3 @@
-VERSION=2021.08.10-0
-SATURNBASE_GPU_IMAGE=saturnbase-gpu-11.2:2021.08.10-0
-IMAGE=saturncloud/saturn-rapids:2021.08.10-0
+VERSION=2021.11.10-1
+SATURNBASE_GPU_IMAGE=saturnbase-gpu-11.2:2021.11.10
```

Comment:
@forana does this need to be prefixed with saturncloud/

Reply:
Oh good catch - looks like that was lost in https://github.com/saturncloud/images/commit/4d5c7384c5397f0b0c9de19215b61275013a6889#diff-9d7f517f941e298a74166113fd26a2ae6e7a0e7e8c1cd50a17c2c5d9801caab7 . Should be automatically corrected the next time these are generated (assuming we fix the automation bug by then).